client/asset/interface.go

```diff
@@ -41,6 +41,9 @@ type WalletInfo struct {
 	// description for each option. This can be used to request config info from
 	// users e.g. via dynamically generated GUI forms.
 	ConfigOpts []*ConfigOption `json:"configopts"`
+	// BuiltIn is true for built-in wallets. Built-in wallets must be created
+	// with a deterministic seed using (Driver).Create before it can be opened.
+	BuiltIn bool
```

Comment:
We're having trouble ironing this out because we're mixing up the purpose in _code_ with what the _user_ should know and see.  This flag is used by Core/consumers to know when to use `CreateWallet` (in Core's case by making a seed and dummy password, etc for `CreateWalletParams`).  In this regard both "created" and "seeded" would be fine.

But TBH I'm pretty darn apathetic about the variable name at this point.  I only want to convey to the user in errors and logs that this is talking about a wallet created and managed by DEX, not that it is "seeded".  Since most external wallets are seeded too, such messages will be super confusing.

So, we can change the variable back as `Seeded` that's fine, let's just decide and move on.  We can change the variable names any time to match changing semantics, but presently this actually creates a wallet stored at the provided DataDir with the provided Seed.

I don't care so please clarify @buck54321

Reply:
> This flag is used by Core/consumers to know when to use CreateWallet.

Not to push it and certainly not a reason to hold back any of the PRs because of this variable name (as long as the user-facing messages are clear and user-friendly), but I do want to add that if this is the purpose of the flag, then `Seeded` doesn't do it for me. As I said in a previous comment, what comes to mind when I see Seeded is, this wallet was created with a seed, not that it even needs to be created with a seed. Maybe that's just me though.